### PR TITLE
Tweaks resulting from "sanity check"

### PR DESCRIPTION
**Pull request by ahemphill**

### CODE REVIEW

File: tools/scoring/score/dimensions/standard_field_naming.py [hunk 1]
```diff
@@ -37,25 +37,25 @@ def _split_subfields(self, field):
...
-    for solution_field, solution_value in solution_translations:
-      solution_subfields = self._split_subfields(solution_field)
-      correct_ceiling += len(solution_subfields)
+    for entity_translations in self.translations.values():
+      for solution_field, solution_value in entity_translations[SOLUTION]:
+        solution_subfields = self._split_subfields(solution_field)
+        correct_ceiling += len(solution_subfields)
 
-      proposed_subfields = set()
+        proposed_subfields = set()
```
Comment (charbull) [inline tools/scoring/score/dimensions/standard_field_naming.py:49]: indentation ok?

File: tools/scoring/score/parse_config.py [hunk 2]
```diff
@@ -149,6 +149,8 @@ def retrieve_reporting_translations(
     for solution_entity in solution_entities.values():
       if solution_entity.cloud_device_id is None:
         continue  # as this is not a reporting device
+      if not Dimension.is_entity_canonical(solution_entity):
```
Comment (charbull) [inline tools/scoring/score/parse_config.py:152]: is this for virtual entities?

**Review by charbull — APPROVED** [review]
questions :)

File: tools/scoring/score/dimensions/standard_field_naming.py (hunk 1 above)
Comment (ahemphill) [reply tools/scoring/score/dimensions/standard_field_naming.py:49]: Yes, everything got bumped over due to `:44` (which was, mercifully, the simple fix).

File: tools/scoring/score/parse_config.py (hunk 2 above)
Comment (ahemphill) [reply tools/scoring/score/parse_config.py:152]: No, it's just an explicit check that simple dimensions are scoring only _canonical_ reporting entities. The complex dimensions invoke it independently.

**Pull request merged**